scripts/downloadGitHubRelease.mjs

```diff
@@ -0,0 +1,118 @@
+import fs from 'fs'
+import path from 'path'
+import { fileURLToPath } from 'url'
+
+const config = {
+  token: process.env.GITHUB_TOKEN,
+  owner: 'fingerprintjs',
+  repo: 'fingerprintjs-pro-cloudflare-worker',
+}
+
+const dirname = path.dirname(fileURLToPath(import.meta.url))
```

Comment:
Suggestion: rename `dirname` to `currentDirName`

Reply:
The `dirname` is based on `__dirname` from CommonJS which is not available here, I think that it makes more sense. But I can change if if you insist 😁

Reply:
When reading the code `path.join(dirname, asset.name)` it is not immediately obvious what `dirname` is but it is not that important, we can keep it.